src/ontology/mondo-ingest.Makefile

```diff
@@ -511,26 +511,36 @@ slurp-all: $(foreach n,$(ALL_COMPONENT_IDS), slurp-$(n))
 sync: sync-subclassof
 
 .PHONY: sync-subclassof
-sync-subclassof: $(REPORTDIR)/sync-subClassOf.direct-in-mondo-only.tsv
+sync-subclassof: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n).subclass.confirmed.robot.tsv)
+	@rm -f $(REPORTDIR)/sync-subClassOf.confirmed.tsv
+	@rm -f $(REPORTDIR)/sync-subClassOf.direct-in-mondo-only.tsv
+	@rm -f $(TMPDIR)/sync-subClassOf.added.self_parentage_issues.tsv
```

Comment:
This is a bit idiosyncratic (using rm), but I guess it safes you from using `-B` which could cause a cascade of updates.

Reply:
Yeah, I suppose I should have added a comment about this. I'm struggling to remember any reason why I did this other than what you mention.

<details><summary>Trying to remember</summary>
<p>

When I think about it, I can only think of me forgetting in the moment that when prereqs are newer than the target, it will rebuild the target, so this forces them to be rebuilt without running `-B`. Or, this might have been because I was originally building the parentage filed by appending to it each time I ran the subclass sync file on a given source, so I needed to remove it first, and I set up the pattern here to be the same for all 3 files. I've since changed it anyway; no longer doing the append thing.

</p>
</details> 

And also, maybe that original append design (which I'm no longer doing) reason is why I changed this goal so that these 3 things are inside the body rather than prereqs. I've just changed it to this:

```make
sync-subclassof: $(REPORTDIR)/sync-subClassOf.confirmed.tsv $(REPORTDIR)/sync-subClassOf.direct-in-mondo-only.tsv $(TMPDIR)/sync-subClassOf.added.self-parentage.tsv
```